GitHub-trainings/git-github-ttl-intro/template.qmd

```diff
@@ -0,0 +1,410 @@
+::: {.column width="50%"}
+
+Git can handle them, but only in the same inefficient way as OneDrive/DropBox. Since all commits is stored on your computer, this will eventually be a major issue.
+
+You can set "_**ignore**_" settings that prevents binary files to be saved in the project folder without being added to Git. We have a [template](https://github.com/worldbank/dime-github-trainings/tree/main/GitHub-resources/DIME-GitHub-Templates) for this settings that can be modified for each project's need.
+
+:::
+
+::: {.column width="50%"}
+TODO: Screen shot of ignore template
+:::
+::::
+
+## Data
+
+:::: {.columns .v-center-container}
+::: {.column width="50%"}
+
+What about data? Most data formats are binary. For example: `.dta` or `.rdata`.
+
+`.csv` is a format that is raw text, but unless it is fully public data that we have right to republish, we should not store any data on GitHub
+
+Git and GitHub is for code, not data.
+
+:::
+
+::: {.column width="50%"}
+TODO: Something
+:::
+::::
+
+## So, where to put data?
+
+```{=html}
+<table style="table-layout: fixed; width: 100%;">
+  <thead>
+    <tr>
+      <th style="width: 33.33%; text-align: center;">API</th>
+      <th style="width: 33.33%; text-align: center;">Manually add</th>
+      <th style="width: 33.33%; text-align: center;">Multi-roots</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td style="width: 33.33%; padding: 25px;">
+        <div style="text-align: justify;">
+          If your data comes from an API, write an API call that loads the data into the code.<br><br>If code saves API data to the Git folder, make sure that data folders or data formats are ignored.
+        </div>
+      </td>
+      <td style="width: 33.33%; padding: 25px;">
+        <div style="text-align: justify;">
+          If the data does not update frequently, it can be shared in any traditional way and the user manually copies the data to the project folder before running the code the first time.<br><br>Make sure that data folders or data formats are ignored.
+        </div>
+      </td>
+      <td style="width: 33.33%; padding: 25px;">
+        <div style="text-align: justify;">
+          Have multiple roots in your projects file paths. Keep data in the old location and then point data file paths to a data folder in a that location. Point code file paths to the Git folder.<br><br>If intermediate data is saved to the Git folder, make sure that data folders or data formats are ignored.
+        </div>
+      </td>
+    </tr>
+  </tbody>
+</table>
+```
+
+# Reproducibilty featueres
+
+## Version control raw text outputs
+
+
+:::: {.columns .v-center-container}
+::: {.column width="50%"}
+
+Tables in `.tex`, `.csv`, or `.md` format works perfectly to version control on Git
+
+This allows you to be fully aware of if any changes in your code makes any difference to your outputs
+
+:::
+
+::: {.column width="50%"}
+TODO: Screen shot of latex table with outputs that has changed
```

Comment:
would be also nice to add a screenshot of a PNG image comparison. That shows a nice feature of GH desktop that would be attractive for TTLs

Reply:
done in d15cfd0449be0eed47359fca473aeefb1967d744